tests/queries/0_stateless/03443_reconnect_n_queries.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+$CLICKHOUSE_BENCHMARK --iterations=10 --reconnect=2 <<< 'SELECT 1' 2>&1 | grep -F 'Queries executed' | tail -n1
```

Comment:
@sachinkumarsingh092 I tested this locally by logging before we disconnect and grepping for that in the log and it looks like we disconnect for every query. For example, for the following query I would expect `disconnecting` to appear 5 times in total, but I get 10 or am I missing something?. Note, I'm running with concurrency=1 (the default value).
```
./clickhouse benchmark --concurrency=1 --iterations=10 --reconnect=2 <<< 'SELECT 1' | grep 'disconnecting' | wc -l
...
10
```
Similar behavior other values.

Reply:
I see the issue, L220 still uses `bool` as the type for reconnect, it should be `size_t`: https://github.com/ClickHouse/ClickHouse/pull/79465/files#diff-6599e8348a85b4c535c41b064f7e5ad25e86d96e0b30ef9641e10092b63a9983L220

Reply:
Thanks for making the changes 🙇